x/auth/client/cli/tx_multisign.go

```diff
@@ -195,6 +201,155 @@ func makeMultiSignCmd() func(cmd *cobra.Command, args []string) (err error) {
 	}
 }
 
+func GetMultiSignBatchCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:    "multisign-batch [file] [name] [[signature-file]...]",
+		Short:  "Assemble multisig transactions",
+		Long:   `Assemble a batch of multisig transactions generated by batch sign command.`,
```

Comment:
This will only work if all signers in all transactions use SIGN_MODE_AMINO_JSON, right? If so, what do you think of making it explicit? Probably the multisign command should also reflect that.

Also, should https://github.com/cosmos/cosmos-sdk/issues/8141 be tackled first? Ideally, we could figure out the same CLI UX for both sign modes (if possible).

Reply:
We can have this to support existing multisig functionality for batch transactions, this helps users using signbatch for multisig transactions.

#8141 is out of scope for stargate, but having this in stargate should be fine imo. wdyt?

Reply:
@amaurymartiny conclusions of #8141 could always be integrated into this command at a later stage I suppose

Reply:
Resolving this. Please reopen if you are unhappy with the agreement reached.